src/Rules.Framework/Builder/Validation/RuleValidator.cs

```diff
@@ -24,5 +27,18 @@ public RuleValidator()
                 ValueConditionNodeValidator = this.valueConditionNodeValidator
             }));
         }
+
+        public static RuleValidator<TContentType, TConditionType> Intance
+        {
+            get
+            {
+                if (ruleValidator is null)
```

Comment:
This if could be replaced with
`return ruleValidator ??= new RuleValidator<TContentType, TConditionType>();`

Reply:
Done